Library/Face.cs

```diff
@@ -34,10 +38,13 @@ public List<Vertex> NeighborVertices()
             return verts;
         }
 
-        /**
-         * Assuming the vertex is part of the face, return the loop such that
-         * loop.vert = v. Return null otherwise.
-         */
+        /// <summary>
+        /// Returns the <see cref="BMeshLib.Loop"/> in the <see cref="Face"/> whose <see cref="Loop.vert"/> matches the specified <see cref="Vertex"/>.
```

Comment:
why is it needed to prefix `Loop` with `BMeshLib.` and not the others?

Reply:
> why is it needed to prefix `Loop` with `BMeshLib.` and not the others?

Because the method is also called `Loop` so the doc comment will refer to the method instead of the class if not specified.